backend/app/helpers/spree/admin/store_credit_events_helper.rb

```diff
@@ -0,0 +1,45 @@
+module Spree::Admin::StoreCreditEventsHelper
+  def admin_action_name(store_credit_event)
+    if Spree::StoreCreditEvent::NON_EXPOSED_ACTIONS.include?(store_credit_event.action) ||
+      store_credit_event.action == Spree::StoreCredit::VOID_ACTION
+      Spree.t("store_credit.display_action.admin.#{store_credit_event.action}")
+    else
+      store_credit_event.display_action
+    end
+  end
+
+  def originator_link(store_credit_event)
```

Comment:
Helpers are exposed globally. Names need to reflect what they are operating on. This should probably be `store_credit_event_originator_link`